src/redux/Missions/Rockets/Rockets.js

```diff
@@ -0,0 +1,44 @@
+const ADD_ROCKET = 'space-travelers-hub/rockets/ADD_ROCKET';
+const RESERVE_ROCKET = 'space-travelers-hub/rockets/RESERVE_ROCKET';
+const initialState = [];
+export const addRocket = (payload) => ({
+  type: ADD_ROCKET,
+  payload,
+});
+export const reserveRocket = (payload) => ({
+  type: RESERVE_ROCKET,
+  payload,
+});
+const rocketReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_ROCKET:
+      return [...state, action.payload];
+    case RESERVE_ROCKET:
+      return state.map((next) => {
+        if ((next.id === action.payload)) {
+          if (next.reserved) {
+            return ({
+              id: next.id,
+              rocket_name: next.rocket_name,
+              description: next.description,
+              image: next.image,
+              wiki: next.wiki,
+              reserved: false,
+            });
+          }
+          return ({
+            id: next.id,
+            rocket_name: next.rocket_name,
+            description: next.description,
+            image: next.image,
+            wiki: next.wiki,
+            reserved: true,
+          });
+        }
+        return next;
+      });
+    default:
+      return state;
+  }
+};
+export default rocketReducer;
```

Comment:
- [ ] I think this file is duplicated as I can see it inside the redux Rockets directory also. Please let's remove this duplicate and leave only the one in the right directory which is `src/redux/Rockets/Rockets.js ` 😄